src/api/spec/features/webui/projects_spec.rb

```diff
@@ -3,6 +3,11 @@
 RSpec.feature "Projects", :type => :feature, :js => true do
   let!(:user) { create(:confirmed_user, login: "Jane") }
 
+  it_behaves_like 'user tab' do
+    let(:project_path) { project_show_path(project: user_tab_user.home_project_name) }
```

Comment:
That's just copied and pasted from line 37-39. The variable is used by the shared example